modules/delegates/components/DelegateDetail.tsx

```diff
@@ -27,11 +27,22 @@ import { useAccount } from 'modules/app/hooks/useAccount';
 import { useActiveWeb3React } from 'modules/web3/hooks/useActiveWeb3React';
 import AccountComments from 'modules/comments/components/AccountComments';
 import { Address } from 'modules/address/components/Address';
+import { formatDelegationHistory } from '../helpers/formatDelegationHistory';
 
 type PropTypes = {
   delegate: Delegate;
 };
 
+const swrPostProcess = useSWRNext => (key, fetcher, config) => {
```

Comment:
what is the use of this exactly? I don't understand it

Reply:
SWR has middleware you can add to pre-process or post-process the data. I thought it would be a good place to add post processing of the delegate history data (so we can process the table and the recharts separately). Just an experiment, I don't know if it adds any benefit vs. just post-processing the data outside of the useSWR hook like we normally do 🤷